homeassistant/components/tankerkoenig/config_flow.py

```diff
@@ -29,6 +29,24 @@
 from .const import CONF_FUEL_TYPES, CONF_STATIONS, DEFAULT_RADIUS, DOMAIN, FUEL_TYPES
 
 
+async def async_get_nearby_stations(
+    hass: HomeAssistant, data: dict[str, Any]
```

Comment:
If we type `data` as `Mapping` we can probably avoid copying the config entry data to a new dict before passing it here.

Reply:
that was to easy 🙈